spec/models/category_spec.rb

```diff
@@ -40,18 +40,18 @@
   end
 
   describe 'associations' do
-    it { expect(subject).to belong_to(:subsection) }
-    it { expect(subject).to have_many(:videos) }
-    it { expect(subject).to have_many(:series) }
+    it { is_expected.to belong_to(:subsection) }
+    it { is_expected.to have_many(:videos) }
+    it { is_expected.to have_many(:series) }
   end
 
   describe 'validations' do
-    it { expect(subject).to validate_presence_of(:subsection) }
-    it { expect(subject).to validate_presence_of(:name) }
-    it { expect(subject).to validate_presence_of(:description) }
+    it { is_expected.to validate_presence_of(:subsection) }
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:description) }
   end
 
-  describe 'latest video' do
+  describe 'latest_video' do
```

Comment:
This should be `describe '#latest_video' do`, shouldn't it? What does rubocop-rspec has to say about it?